app/api_topology_test.go

```diff
@@ -54,27 +54,6 @@ func TestAll(t *testing.T) {
 	}
 }
 
-func TestAPITopologyContainers(t *testing.T) {
-	ts := topologyServer()
```

Comment:
I dropped this test as rendered containers contain a bunch of extra special casing (image names in rank etc) and its verbose testing that doesn't add anything.